src/actions/clusterActions.js

```diff
@@ -245,10 +245,14 @@ export function clusterLoadApps(clusterId) {
     return appsApi
       .getClusterApps(clusterId)
       .then(apps => {
+        // For some reason the array that we get back
+        // from the generated js client does not have
+        // .map on it. So I make a new one here.
+        const appsArray = apps.map ? apps : Array.from(apps);
```

Comment:
The actual explanation would be that the value returned is an [*Iterable* (or an *Array-Like* data structure)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Iteration_protocols) which does not have methods specific to arrays (such as `forEach` or `map`). Hence the need for creating an array out of it.

Since `Array.from()` also works on arrays, can we just keep `Array.from(apps)`? (to decrease complexity and for easier understanding later on)

Reply:
I'd be happy with just Array.from() instead of the ternary.